### PR TITLE
encryption=on|off option for 'create service' cmd

### PR DESCRIPTION
**Pull request by rentallect**
nothing to see here... move along...

### CODE REVIEW

File: ziti/cmd/ziti/cmd/edge_controller/create_service.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ type createServiceOptions struct {
 	roleAttributes     []string
 	configs            []string
 	requireEncryption  bool
-	optionalEncryption bool
+	encryption         encryptionVar
```
Comment (dovholuknf) [inline ziti/cmd/ziti/cmd/edge_controller/create_service.go:36]: shouldn't the changelog get updated too?

File: ziti/cmd/ziti/cmd/edge_controller/encryption_var.go [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	value = strings.ToUpper(value)
+	if value != "ON" && value != "OFF" && value != "TRUE" && value != "FALSE" {
+		return errors.New("Invalid option -- must specify either 'ON' or 'OFF'")
+	}
+	*f = encryptionVar(value)
+	return nil
+}
+
+func (f *encryptionVar) Get() bool {
+	if string(*f) == "ON" || string(*f) == "TRUE" {
```
Comment (dovholuknf) [inline ziti/cmd/ziti/cmd/edge_controller/encryption_var.go:41]: personally i think this should normalize the input to upper or lower too... but maybe i'm just overly cautious? :)

**Review by dovholuknf — APPROVED** [review]

File: ziti/cmd/ziti/cmd/edge_controller/encryption_var.go (hunk 2 above)
Comment (ekoby) [reply ziti/cmd/ziti/cmd/edge_controller/encryption_var.go:41]: yes, no reason to require capital letters

Comment (rentallect) [reply ziti/cmd/ziti/cmd/edge_controller/encryption_var.go:41]: We do not require caps at the cmd line. The user can enter lower, upper, or mixed case for the on|off|true|false value.  This up-casing is just to make internal comparison easy.

File: ziti/cmd/ziti/cmd/edge_controller/create_service.go (hunk 1 above)
Comment (andrewpmartinez) [reply ziti/cmd/ziti/cmd/edge_controller/create_service.go:36]: I second this.

**Review by andrewpmartinez — CHANGES_REQUESTED** [review]
See Clint's message about the change log.

File: ziti/cmd/ziti/cmd/edge_controller/create_service.go (hunk 1 above)
Comment (rentallect) [reply ziti/cmd/ziti/cmd/edge_controller/create_service.go:36]: done

**Review by andrewpmartinez — APPROVED** [review]

**Pull request merged**